### PR TITLE
Joe-submission-info command Update

### PR DESCRIPTION
**Pull request by AradCarmi**
## Description
The `submission_ids` argument of the `joe-submission-info` command is now required because the command's purpose is to retrieve information for a specific ID. Without this argument, the command cannot function as intended. This design ensures that the necessary information is not overlooked.

### CODE REVIEW

File: Packs/JoeSecurity/ReleaseNotes/1_1_21.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Joe Security v2
+
+- Updated the **joe-submission-info** command to requires the *submission_ids* argument to be provided.
```
Comment (Shellyber) [inline Packs/JoeSecurity/ReleaseNotes/1_1_21.md:6]:
```suggestion
- The *submission_ids* argument is now required when using the command **joe-submission-info**.
```

**Review by Shellyber — COMMENTED** [review]
Nice. see the rn change

**Review by shmuel44 — APPROVED** [review]
Nice!

**Review by Shellyber — APPROVED** [review]
Nice :)

**Comment by AradCarmi** [comment]
Force merge is needed as from now the `submission_ids` argument of the  `joe-submission-info` command is required.

**Pull request merged**